bitcoind-rpc/src/main/scala/org/bitcoins/rpc/client/common/BitcoindRpcClient.scala

```diff
@@ -118,8 +119,9 @@ class BitcoindRpcClient(val instance: BitcoindInstance)(implicit
 
   // Node Api
 
-  override def broadcastTransaction(transaction: Transaction): Future[Unit] =
-    sendRawTransaction(transaction).map(_ => ())
+  override def broadcastTransactions(
+      transactions: Vector[Transaction]): Future[Unit] =
+    FutureUtil.sequentially(transactions)(sendRawTransaction(_)).map(_ => ())
```

Comment:
nit: I don't see why we can't broadcast these in parallel? Is there some sort of rate limit with bitcoin core?

Reply:
The RPC has a max of 16 requests at at time, then it stops dropping them